### DISCUSSION TITLE
Save Filters As Presets in Database

### DISCUSSION THREAD

**Discussion by BenOussama180** [Power-Components/livewire-powergrid#491] [Ideas]
So i have a table with multiple filters, i want to save some filter presets in database and get them on click of a button and refresh table so those filters would be applied, i know there is a feature or "persist" so this can be done 
any idea on hiw i can achieve that ? saving filters in database

**Comment by luanfreitasdev** [comment]
Have you tried using the updatedFilters method and saving the content in the database?

**Reply by BenOussama180** [reply]
no , i had no clue such function existed, i will give it a try ,thanks

**Reply by luanfreitasdev** [reply]
Using the filters property lifecycle
https://laravel-livewire.com/docs/2.x/lifecycle-hooks

**Reply by BenOussama180** [reply]
oooh thats pretty clever, thanks for the tip
